helm/flink-kubernetes-operator/templates/flink-operator.yaml

```diff
@@ -130,6 +141,10 @@ spec:
             - name: flink-artifacts-volume
               mountPath: /opt/flink/artifacts
             {{- end }}
+          {{- if .Values.tls.create }}
+            - name: flink-operator-cert-secret
+              mountPath: /opt/flink/artifacts/operator-cert
```

Comment:
I don't think we can mount under `/opt/flink/artifacts` , this should not overlap and be something like `/opt/flink/certs` instead